docs/settingUp.md

```diff
@@ -88,9 +88,14 @@ Eclipse IDE is our preferred development environment. Support requests related t
       ![devverification-guide-1.png](images/devverification-guide-1.png)
      
    * Indentation: In TEAMMATES, we use 4 spaces in place of tabs for indentations.
-     Go to `Eclipse → Preferences` and configure for all the languages used in TEAMMATES:
-     * Java: `Java → Code Style → Formatter → Edit → Tab policy → Spaces only`
-     * JavaScript: `JavaScript → Code Style → Formatter → Edit → Tab policy → Spaces only`
+   * Trailing whitespace: all forms of trailing whitespace is disallowed, blank lines included.
+   * Go to `Eclipse → Preferences` and configure for all the languages used in TEAMMATES:
+     * Java:
+        * `Java → Code Style → Formatter → Edit → Tab policy → Spaces only`
+        * `Java → Editor → Save Actions → Additional actions → Configure → Remove trailing whitespace → All lines`
```

Comment:
> The other 3/4 actions, however, are up to the developer. I wouldn't want to even mention them in the instructions.

Agree entirely on this. Some things are best not to be mentioned unless we're creating a "Eclipse best practices" document or alike. Unfortunately, Eclipse puts trailing whitespace on the same category as them and I don't think we can really single out the trailing whitespace fix without making mentions to other actions. It's not that those actions are unfavourable (in fact my own Eclipse settings have them all and beyond), but we shouldn't cross to the level of micro-managing because those are entirely up to the developers.